src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -20,6 +20,10 @@
   </ItemGroup>
 
   <PropertyGroup>
+    <TestWithCore Condition="'$(TestWithCore)' == '' And '$(TargetGroup)' == 'net45'">false</TestWithCore>
```

Comment:
We should consider moving these propertyes and the TestFramework ItemGroup below into dir.props so that we can try to keep the TargetGroup conditions localized there and setup the specifics in a common location.